presto-hive/pom.xml

```diff
@@ -191,16 +191,6 @@
             <artifactId>javax.inject</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-core</artifactId>
-        </dependency>
```

Comment:
How compile scope dependencies are related to test removal?

Reply:
The test was the only remaining usage. The dependency analyzer doesn't complain if you have a compile dependency that is only used by test code.